web/index.php

```diff
@@ -101,13 +101,19 @@
 $app->container->set('access_token', isset($_SESSION['access_token']) ? $_SESSION['access_token'] : null);
 
 $app->container->singleton(\Application\CacheService::class, function ($container) {
-    $redisServer = null;
+
+    $redis = $container->settings['custom']['redis'];
+
     if ($host = getenv('REDIS_HOST')) {
-        $redisServer = "tcp://$host:6379";
+        $redis['scheme'] = 'tcp';
+        $redis['host'] = $host;
+        $redis['port'] = 6379;
     }
 
-    $client = new Predis\Client($redisServer);
-    $keyPrefix = $container->settings['custom']['redisKeyPrefix'];
+    $keyPrefix = $redis['keyPrefix'];
+    unset($redis['keyPrefix']);
```

Comment:
Unsetting $redis['keyPrefix'] here feels wrong is we have to unset it here, we shouldn't set it in this array in the first place

Reply:
I went ahead and removed the unset. I don't think it's going to hurt anything anyway.